package/TSAccountProtectionProvider.tsx

```diff
@@ -17,14 +17,20 @@ const SDK_TRIGGER_ACTION_ERR = 'Error sending action event';
 const SDK_AUTHENTICATE_USER_ERR = 'Error authenticating user';
 const SDK_CLEAR_USER_ERR = 'Error clearing user';
 
-type ProviderState = {
+type ProviderState = DRSConfigOptions & {
   initialized: Promise<boolean>;
+};
+
+type ErrHandler = (err: any) => void;
+
+export type DRSConfigOptions = {
```

Comment:
please add to this type similar documentation to what we have in angular's [config object](https://github.com/TransmitSecurity/ngx-ts-riskid/blob/main/projects/ngx-ts-riskid/src/lib/ngx-ts-riskid.service.ts#L8) so that it's clear how to use it

Reply:
done